server/routes/products.js

```diff
@@ -0,0 +1,6 @@
+const router = require('express').Router();
+const { fetchProducts } = require('../controllers');
+
+router.get('/products', fetchProducts);
```

Comment:
don't make sub-directories for products routes, simply call your route inside `routes/index.js`